libtmux/test.py

```diff
@@ -43,6 +46,52 @@ def retry(seconds=RETRY_TIMEOUT_SECONDS):
     return (lambda: time.time() < time.time() + seconds)()
 
 
+def retry_until(
+        fun,
+        seconds=RETRY_TIMEOUT_SECONDS,
```

Comment:
```suggestion
        seconds: float=RETRY_TIMEOUT_SECONDS,
```

Based on [`time.sleep()`](https://github.com/python/typeshed/blob/740193a/stdlib/time.pyi#L76) annotation